### ISSUE TITLE
Feature Request: Link to comments in Files app of Nextcloud and Owncloud

### ISSUE THREAD

**Issue by sunjam** [pbek/QOwnNotes#776]
I'd like to link and communicate between Qownnotes and the comments users can attach in the Files app of Owncloud and Nextcloud.  Not sure how this is possible, but it'd be an excellent feature.

**Comment by pbek** [comment]
Can you please elaborate a bit on your idea.

**Comment by sunjam** [comment]
Sure.  I'm wondering how to get Finder App file comments to synchronize with Qownnotes as they currently do with the Activity app... here is an example of what I see under the Activity App once I leave a comment.
![screenshot_2017-11-20_12-52-20](https://user-images.githubusercontent.com/1787238/33040998-b15f8aac-cdf1-11e7-808e-00e9851ff312.png)
How would I get this to be entered in my notes app as well?  [Just googled this thread](https://help.nextcloud.com/t/info-about-activities-other-than-e-mail/11286/6) for more info on how Activity App shows information to users.    
    
Ideally, I could see the comment's content, date and time stamp, user name, and a link to the file mentioned within Qownnotes App.

**Comment by pbek** [comment]
QOwnNotes doesn't really trigger activities because the syncing is done by the oC sync client and not by QOwnNotes...

Why do you want to see comments of note-files inside QOwnNotes?

**Comment by sunjam** [comment]
Yes, I’d love to see nextcloud comments of note-files inside of Qownnotes.  Perhaps they could be viewed as an annotation.
